packages/nocodb/docker/start-litestream.sh

```diff
@@ -19,14 +26,16 @@ use_litestream() {
   && [ -z "${DATABASE_URL}" ] \
   && [ -z "${DATABASE_URL_FILE}" ] \
   && [ -z "${NC_MINIMAL_DBS}" ] \
-  && [ -n "${LITESTREAM_S3_ENDPOINT}" ] \
   && [ -n "${LITESTREAM_S3_BUCKET}" ] \
   && [ -n "${LITESTREAM_S3_ACCESS_KEY_ID}" ] \
   && [ -n "${LITESTREAM_S3_SECRET_ACCESS_KEY}" ]
 }
 
 if use_litestream ; then
 
+  # set default bucket path if not provided
+  : "${LITESTREAM_S3_PATH:=nocodb}"
```

Comment:
To be consistent, I would rather set this in the [`Dockerfile` where the other Litestream-specific defaults are set](https://github.com/nocodb/nocodb/blob/f441403abadb91ad3d07cd9dd943b626777a8571/packages/nocodb/Dockerfile#L53).

Reply:
If we set it at docker file it's hard to add backward compatibility for env var `AWS_BUCKET_PATH`